src/transforms/tree.d.ts

```diff
@@ -2,14 +2,106 @@ import type {ChannelValue} from "../channel.js";
 import type {CompareFunction, Transformed} from "./basic.js";
 
 export interface TreeTransformOptions {
+  /**
+   * A channel specifying each node’s hierarchy location. The **path** channel
+   * is typically slash-separated, as with UNIX-based file systems or URLs.
+   * Defaults to identity.
+   */
   path?: ChannelValue;
+  /**
+   * The path separator; defaults to forward slash (/).
+   */
   delimiter?: string;
+  /**
+   * If the **treeAnchor** is *left*, the root of the tree will be aligned with
```

Comment:
These options could use a summary sentence prior to diving into the possible ways in which the options could be specified. For example:

* treeAnchor - how to orient the tree
* treeLayout - how to position nodes in the tree
* treeSeparation - how much space to reserve between adjacent nodes
* treeSort - how to order nodes prior to computing the layout

etc.